src/docs/user/references/Tasks.adoc

```diff
@@ -1,4 +1,10 @@
-This section describes the available predefined tasks in the *Tasks* menu of *ProActive Studio* and the operating systems where they run.
+This section describes the available predefined tasks in the *Tasks* menu of *ProActive Studio* and the operating systems where they run. These tasks are grouped in three sections:
+
+1. _OS Shells_: Shell, Bash, Windows Cmd, PowerShell
+2. _Languages_: Java, Scalaw, Javascript, Groovy, Ruby, Jython, Python, Perl, PHP, R
+3. _Containers_: Docker File, Docker Compose, Kubernetes
```

Comment:
Docker File => Dockerfile

Reply:
In the Tasks menu, it is rather Docker_File and Docker_Compose

Reply:
Then u might use`Docker_File`

Reply:
even though Docker_File is not correct and should be `Dockerfile` in the studio and in the doc

Reply:
Dac